BridgeAppSDK/NSDate+Utilities.swift

```diff
@@ -41,5 +41,13 @@ extension NSDate {
         let components = calendar.components(unitFlags, fromDate: self)
         return calendar.dateFromComponents(components) ?? self
     }
+    
+    var isToday: Bool {
+        return self.startOfDay() == NSDate().startOfDay()
+    }
+    
+    var isTomorrow: Bool {
+        return self.startOfDay() == NSDate(timeIntervalSinceNow: 24 * 60 * 60).startOfDay()
```

Comment:
We've done this in a number of places already (1 day == 86400 seconds), but it turns out it isn't safe; it's wrong for an hour on changes between Standard and Daylight time and back (and for 1 second if there's a leap second, but that's hardly a problem and pretty rare anyway). I think the "right" way is to use date components and add one day. We should fix these in our code as we find them.